apps/examples/wifi_manager_sample/wm_test_stress_test1.c

```diff
@@ -66,7 +68,7 @@ static wifi_manager_cb_s g_wifi_callbacks = {
 	NULL, NULL, NULL
```

Comment:
NULL values can be replaced with wm_softap_sta_joined, wm_softap_sta_left, wm_scan_ap_done